test/cluster_commands_on_strings_test.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+require_relative 'helper'
+
+# ruby -w -Itest test/cluster_commands_on_strings_test.rb
+# @see https://redis.io/commands#string
+class TestClusterCommandsOnStrings < Test::Unit::TestCase
+  include Helper::Cluster
+
+  def test_append
```

Comment:
It seems to me that many of these added teste are not really specific to cluster mode.

Some do assert on the `CROSSSLOT Keys` error, but many would work just fine on a regular Redis.

There is already shared suites applied on both `Redis` and `Redis::Distributed`, see `Lint::String` for example. 

Using these shared tests should allow to reduce the number of added tests, as well as reduce the duplication, so less risks of forgetting about clusters when new commands etc are added in the future.